types/fancy-log/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for fancy-log 1.3
+// Project: https://github.com/js-cli/fancy-log
+// Definitions by: Pine Mizune <https://github.com/pine>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace FancyLog {
+    interface Logger {
+        (...args: any[]): Logger;
+        dir(...args: any[]): Logger;
+        error(...args: any[]): Logger;
+        info(...args: any[]): Logger;
+        warn(...args: any[]): Logger;
+    }
+}
+
+declare var logger: FancyLog.Logger;
```

Comment:
Doesn't seem like there's much of a reason not to just define `logger` directly as a function but lgtm.